lib/merge.ts

```diff
@@ -1,8 +1,15 @@
 import mergeWith from 'lodash.mergewith';
 
 export const merge = mergeWith;
-export const mergeCustomizer = (_object: any, srcVal: any) => {
+export const mergeCustomizer = (
+  _object: any,
```

Comment:
This is partially my bad because I named this argument initially, but we should probably rename it to `objValue` to match the [Lodash documentation](https://lodash.com/docs/4.17.15#mergeWith), and it no longer needs the `_` prefix since the variable is being used.

Reply:
Modified to use the same naming conventions as per the lodash docs in [commit](https://github.com/thoughtbot/fishery/pull/62/commits/fb875960b7098f7b56fe7c37bb774b098cb578cc)